src/lib/features/download-episode/lib/specs/download.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { get } from 'svelte/store';
+import { startEpisodeDownload } from '../download';
+
+const sleep = (ms: number) => new Promise<void>((resolve) => setTimeout(resolve, ms));
```

Comment:
issue: whenever we're dealing with timeouts, I think we should use fake clocks to prevent flake.

suggestion: see https://github.com/aabounegm/cast/blob/main/src/lib/shared/ui/specs/snackbar.spec.ts#L48